src/lib.rs

```diff
@@ -118,6 +132,12 @@ impl<T: FileLoader> VfsInternal<T> {
         }
         result
     }
+
+    fn get_line(&self, path: &Path, line: usize) -> Option<String> {
+        self.load_file(path);
```

Comment:
it should be the callers responsibility to make sure the file is loaded. If this is hard to track, then we should add an `ensure_file` method which they can call to make sure it is loaded.